core/src/main/kotlin/org/evomaster/core/problem/security/service/SSRFAnalyser.kt

```diff
@@ -135,14 +133,25 @@ class SSRFAnalyser {
             should check the content of rcr result
          */
 
-        val hasCallbackURL = action.parameters.any { param ->
-            val genes = getStringGenesFromParam(param.seeGenes())
-            genes.any { gene ->
+        val hasCallBackURL = GeneUtils
+            .getAllStringFields(action.parameters)
+            .any { gene ->
                 httpCallbackVerifier.isCallbackURL(gene.getValueAsRawString())
             }
+
+        if (hasCallBackURL) {
+            // FIXME: When the code reaches this point during SSRF phase
```

Comment:
Hi @arcuri82,

I can see the code reach the AbstractRestFitness during the SSRF detection phase. However, WireMock seems to be null at this point although it was initiated earlier. I'm not sure why this is happening, need your assistance to debug this further.